x-pack/plugins/spaces/public/views/management/edit_space/manage_space_page.test.tsx

```diff
@@ -108,23 +112,121 @@ describe('ManageSpacePage', () => {
 
     wrapper.update();
 
-    const nameInput = wrapper.find('input[name="name"]');
-    const descriptionInput = wrapper.find('textarea[name="description"]');
+    updateSpace(wrapper);
 
-    nameInput.simulate('change', { target: { value: 'New Space Name' } });
-    descriptionInput.simulate('change', { target: { value: 'some description' } });
-
-    const createButton = wrapper.find('button[data-test-subj="save-space-button"]');
-    createButton.simulate('click');
-    await Promise.resolve();
+    await clickSaveButton(wrapper);
 
     expect(spacesManager.updateSpace).toHaveBeenCalledWith({
       id: 'existing-space',
       name: 'New Space Name',
       description: 'some description',
       color: '#aabbcc',
       initials: 'AB',
-      disabledFeatures: [],
+      disabledFeatures: ['foo'],
     });
   });
+
+  it('warns when updating features in the active space', async () => {
```

Comment:
Should we add a test for when we update the active space and don't change any features?

Reply:
good idea, will do!